api-tests/features/inherited-identity/extended-scenarios.feature

```diff
@@ -0,0 +1,290 @@
+@Build
+Feature: Inherited identity extended scenarios
+  Scenario Outline: Successfully migrates <inherited-identity> HRMC profile after partial P2 journey
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-passport-valid' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+
+    # New journey with inherited identity
+    When I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity '<identity-details>'
+    Then I get an OAuth response
+    And an 'IPV_INHERITED_IDENTITY_VC_RECEIVED' audit event was recorded [local only]
+    When I use the OAuth response to get my identity
+    Then I get a '<expected-identity>' identity
+
+    Examples:
+    | inherited-identity | identity-details             | expected-identity |
+    | PCL200             | alice-vot-pcl200-no-evidence | PCL200            |
+    | PCL250             | kenneth-vot-pcl250-passport  | PCL250            |
```

Comment:
Do you think there's much value running this with different profiles? Feels like one would probably be sufficient...

Same applies throughout this file tbh - the important part of these tests is how inherited IDs behave with other things (CIs, F2F, etc.), rather than the profile itself